WordPress/Classes/ViewRelated/Post/PostEditor+BlogPicker.swift

```diff
@@ -1,102 +0,0 @@
-import UIKit
-
-extension PostEditor where Self: UIViewController {
-
-    func blogPickerWasPressed() {
-        assert(isSingleSiteMode == false)
-        // if it's a reblog, we won't change the content, so don't show the alert
-        guard !postIsReblogged, post.hasSiteSpecificChanges()  else {
-            displayBlogSelector()
-            return
-        }
-
-        displaySwitchSiteAlert()
-    }
-
-    func displayBlogSelector() {
-        guard let sourceView = navigationBarManager.blogPickerButton.imageView else {
-            fatalError()
-        }
-
-        // Setup Handlers
-        let successHandler: BlogSelectorSuccessHandler = { selectedObjectID in
-            self.dismiss(animated: true)
-            WPAnalytics.track(.editorPostSiteChanged)
```

Comment:
Curious about this one, not sure if we should also remove the `editorPostSiteChanged` event? 🤔

Reply:
I think we should unless there is any other way to _switch site_. I think there is none, what do you think.

Reply:
Actually this event is used just for the Post Editor, So we can safely remove it 😄
Fixed at ea9a275

Reply:
Is this event also gone from WPAndroid? I'm not sure if there are extra steps needed to be done when removing an analytics event altogether. Is it possible for you to check maybe if anything else is needed?

Reply:
> Is this event also gone from WPAndroid?

I think this `site switch feature`/event is not there in WPAndroid as the enhancement issue got closed.

> I'm not sure if there are extra steps needed to be done when removing an analytics event altogether. Is it possible for you to check maybe if anything else is needed?

Will double check 💯

Reply:
> Will double check 💯

I think it should be fine as we are tracking just the **string**(value of `WPAnalyticsEvent`) with the default properties in this case.
There is another way of tracking using `WPAnalyticsStat` which is used for _Reader_, not related to our usecase.